build_resume.py

```diff
@@ -9,11 +9,7 @@
 
 
 def generate_filename(last_name=""):
-    date = time.strftime("%Y-%m-%d")
-    return "Resume_{name}{date}".format(
-        name=last_name+"_" if last_name else "",
-        date=date
-    )
+    return "Resume{name}".format(name="_"+last_name if last_name else "")
```

Comment:
on second thought, just remove this function since it's a very readable oneline now.